perceval/converters/qiskit_converter.py

```diff
@@ -84,10 +84,26 @@ def convert(self, qc, use_postselection: bool = True) -> Processor:
             # some limitation in the conversion, in particular measure
             assert isinstance(instruction[0], qiskit.circuit.gate.Gate), "cannot convert (%s)" % instruction[0]
 
+            gate_name = instruction[0].name
```

Comment:
To be changed to `instruction.operation.name`  after rebasing on the release branch after merging of the "Update Qiskit" branch. 
Same for the 4 occurrences below